src/test/java/org/prebid/server/handler/StatusHandlerTest.java

```diff
@@ -50,6 +52,8 @@ public void shouldRespondHttp200OkWithExpectedBody() throws JsonProcessingExcept
         statusHandler = new StatusHandler(Arrays.asList(healthCheck, healthCheck, healthCheck), jacksonMapper);
 
         given(routingContext.response()).willReturn(httpResponse);
+        given(httpResponse.putHeader(HttpUtil.CONTENT_TYPE_HEADER, HttpHeaderValues.APPLICATION_JSON))
```

Comment:
Pls replace with:
```
given(httpResponse.putHeader(any(CharSequence.class), any(AsciiString.class))).willReturn(httpResponse);
```
since mocking conditions can be more generic.